example/src/components/DemoResponse.tsx

```diff
@@ -7,7 +7,7 @@ export function DemoResponse({children}: React.PropsWithChildren<{}>) {
   }
 
   return (
-    <ScrollView style={styles.container}>
+    <ScrollView style={styles.container} nestedScrollEnabled>
```

Comment:
Without this option, the view is not scrollable on Android and you only see the first lines